lib/node_modules/@stdlib/math/base/special/erfcx/src/main.c

```diff
@@ -0,0 +1,1162 @@
+* Chebyshev polynomial for \[0.99, 1.00).
+*
+* @param t   scaled value
+* @return    result
+*/
+static double p99( const double t ) {
+	return 0.97771701335885035464e0 + (0.22000938572830479551e-1 + (0.27951610702682383001e-3 + (0.25153688325245314530e-5 + (0.16514019547822821453e-7 + (0.78191526829368231251e-10 + 0.24873652355555555556e-12 * t) * t) * t) * t) * t) * t;
+}
+
+/**
+* Chebyshev polynomial for `1.0`.
+*
+* @param t   scaled value
+* @return    result
+*/
+static double p100() {
+	// We only get here if y = 1; i.e., |x| < 4*eps, in which case `erfcx` is within `1e-15` of 1...
+	return 1.0;
+}
+
+
+static double (* const table[])(const double) = {
+	p0, p1, p2, p3, p4, p5, p6, p7, p8, p9,
+	p10, p11, p12, p13, p14, p15, p16, p17, p18, p19,
+	p20, p21, p22, p23, p24, p25, p26, p27, p28, p29,
+	p30, p31, p32, p33, p34, p35, p36, p37, p38, p39,
+	p40, p41, p42, p43, p44, p45, p46, p47, p48, p49,
+	p50, p51, p52, p53, p54, p55, p56, p57, p58, p59,
+	p60, p61, p62, p63, p64, p65, p66, p67, p68, p69,
+	p70, p71, p72, p73, p74, p75, p76, p77, p78, p79,
+	p80, p81, p82, p83, p84, p85, p86, p87, p88, p89,
+	p90, p91, p92, p93, p94, p95, p96, p97, p98, p99,
+	p100
+};
+
+/**
+* Given `y100 = 100*y`, where `y = 4/(4+x)` for `x >= 0`, compute `erfc(x)`.
+*
+* ## Notes
+*
+* -   Uses a look-up table of 100 different Chebyshev polynomials or y intervals \[0,0.01\], \[0.01,0.02\], ..., \[0.99,1\], generated with the help of Maple and a little shell script.
+* -   This allows the Chebyshev polynomials to be of significantly lower degree (about 1/4) compared to fitting the whole \[0,1\] interval with a single polynomial.
+*
+* @param y100 - scaled argument
+* @return complementary error function
+*/
+static double erfcxY100( const double y100 ) {
+	double t;
+	double(*f)(const double);
+
+	t = stdlib_base_floor( y100 );
+	f = table[ (int)t ];
+	return f( 2 * y100 - ( ( 2 * t ) + 1)  );
+}
+
+/**
+* Evaluates the scaled complementary error function.
+*
+* ```tex
+* \operatorname{erfcx}(x) = \exp{x^2} \cdot \operatorname{erfc}(x)
+* ```
+*
+* ## Notes
+*
+* -   Use this function to replace expressions containing `exp{x^2} erfc(x)` in order to avoid errors due to underflow or overflow.
+* -   For expressions of the form `\exp{-x^2} erfcx(x)`, use the complementary error function `erfc(x)` instead, as this substitution maintains accuracy by avoiding roundoff errors for large values of `x`.
+*
+* @param x - input value
+* @return evaluated scaled complementary error function
+*
+* @example
+* double y  = stdlib_base_erfcx( 0.0 );
+* // returns 1.0
+*
+* @example
+* double y  = stdlib_base_erfcx( 1.0 );
+* // returns ~0.4276
+*
+* @example
+* double y  = stdlib_base_erfcx( -1.0 );
+* // returns ~5.01
+*
+* @example
+* double y  = stdlib_base_erfcx( 50.0 );
+* // returns ~0.011
+*
+* @example
+* double y  = stdlib_base_erfcx( -50.0 );
+* // returns +Infinity
+*/
+double stdlib_base_erfcx( const double x ) {
+	double x2;
+
+	if( stdlib_base_is_nan(x) ) {
```

Comment:
```suggestion
	if( stdlib_base_is_nan( x ) ) {
```

Reply:
@kgryte Similar to this, should we use `isnan` function in [`main.js`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/erfcx/lib/main.js#L103) as well, rather than checking `x !== x`?

Reply:
Yes, I think that would be reasonable.